Makefile.old

```diff
@@ -62,17 +65,19 @@ clean-db: services-clean-db
 clean-all: clean clean-db
 
 check: test
-test: build ui-test services-test pp-test plugins-test
+test: build services-test pp-test plugins-test
 	bin/HootTest $(HOOT_TEST_DIFF) --slow
 
 docs: always HOOT_VERSION_FILE services-docs
+	echo Building documentation...
 	cd docs; $(MAKE)
 
-test-quick: build services-test pp-test
+test-quick: build pp-test
 	bin/HootTest $(HOOT_TEST_DIFF) --quick
 
-test-all-no-core: ui-test build services-test-all pp-test plugins-test
-
+test-all-no-core: services-test-all pp-test plugins-test
+	make ui-test
```

Comment:
The problem I had was that if I didn't add a separate make command for ui-test at the end of that sequence, then both the services tests and the ui tests would run at the same time, and the machine slows to an unbearable crawl.  I can't make services-test a pre-req for ui-test, b/c that assumes whoever is running ui-test also wants to run services-test, but that may not be true.  If there's a better way in the makefile to prevent services-test and ui-test from running at the same time, let me know.